Lib/test/test_threading.py

```diff
@@ -1338,6 +1339,24 @@ def run(self):
         # explicitly break the reference cycle to not leak a dangling thread
         thread.exc = None
 
+    def test_multithread_modify_file_noerror(self):
+        def modify_file():
+            with open(__file__, 'a') as fp:
+                fp.write(' ')
+                traceback.format_stack()
+
+        threads = [
+            threading.Thread(target=modify_file)
+            for i in range(100)
+        ]
+        try:
+            for t in threads:
+                t.start()
+            for t in threads:
+                t.join()
```

Comment:
Is there a reason why you don't do 
```
for t in threads:
    t.start()
    t.join()
```
?

Reply:
Nothing, sorry.
So i fixed it to your feedback.